### PR TITLE
Text updates to Load Testing

### PR DESCRIPTION
**Pull request by icchan**
I made additional wording updates to the load testing section based on PR review comments that I missed previously.

### CODE REVIEW

**Review by RichCzyzewski — APPROVED** [review]
Looks great!

File: automated-testing/performance-testing/load-testing.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ There are a number of basic component which are required to carry out a load tes
 
 1. In order to have meaningful results the system needs to be tested in a production-like environment with a network and hardware which closely resembles the expected deployment environment.
 
-2. The load will consist of a module which simulates user activity. Of course what the composition of this "user activity" will be vary based on the type of application being tested. For example an e-commerce web site might simulate user browsing and purchasing items, but an IoT data ingestion pipeline would simulate a stream of device readings.
+2. The load test will consist of a module which simulates user activity. Of course what the composition of this "user activity" will be vary based on the type of application being tested. For example an e-commerce web site might simulate user browsing and purchasing items, but an IoT data ingestion pipeline would simulate a stream of device readings. Please ensure the simulation is as close to real activity as possible taking into account not just volume but also patterns and variability. For example if the simulator data is too uniform or predictable then cache/hit ratios may impact your results.
```
Comment (maniSbindra) [inline automated-testing/performance-testing/load-testing.md:20]:
```suggestion
2. The load test will consist of a module which simulates user activity. Of course what the composition of this "user activity" will vary based on the type of application being tested. For example an e-commerce web site might simulate user browsing and purchasing items, but an IoT data ingestion pipeline would simulate a stream of device readings. Please ensure the simulation is as close to real activity as possible taking into account not just volume but also patterns and variability. For example if the simulator data is too uniform or predictable then cache/hit ratios may impact your results.

```

Comment (maniSbindra) [reply automated-testing/performance-testing/load-testing.md:20]: Should "will be vary" be changed to "will vary"

**Review by maniSbindra — APPROVED** [review]
looks good to me, just one minor suggestion

**Comment by TessFerrandez** [comment]
@icchan do you have permissions to merge or do you want me to merge?

**Pull request merged**